src/renderers/webgl/WebGLLights.js

```diff
@@ -117,7 +117,8 @@ function WebGLLights() {
 			spotLength: - 1,
 			rectAreaLength: - 1,
 			hemiLength: - 1,
-			shadowsLength: - 1
+			shadowsLength: - 1,
+			value: 0
```

Comment:
To be consistent with similar code, I would name this property `version`.

Reply:
done, I wasn't happy with that name.